[Bug]: playwright mcp package installation issue -- Cannot find module './lib/servers/snapshot'

### Version

Version 1.42.1

### Steps to reproduce

1.  npm install --save-exact @playwright/mcp@1.52.0-alpha-2025-03-14   -- Looks installation is fine
npm warn deprecated @playwright/test@1.42.1: Please update to the latest version of Playwright to test up-to-date browsers.

added 3 packages, changed 1 package, and audited 96 packages in 3s

16 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities  

2. npx @playwright/mcp --help


### Expected behavior

Expected to run well with no error

### Actual behavior

node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module './lib/servers/snapshot'
Require stack:
- /Users/junp/code/nvcf-ui-test/node_modules/@playwright/mcp/cli.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (/Users/junp/code/nvcf-ui-test/node_modules/@playwright/mcp/cli.js:18:1)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/junp/code/nvcf-ui-test/node_modules/@playwright/mcp/cli.js'
  ]
}

Node.js v22.14.0

### Additional context

_No response_

### Environment

```shell
Darwin junp-mlt 23.6.0 Darwin Kernel Version 23.6.0: Fri Nov 15 15:12:37 PST 2024; root:xnu-10063.141.1.702.7~1/RELEASE_ARM64_T6030 arm64
```

--- a/packages/playwright-mcp/.npmignore
+++ b/packages/playwright-mcp/.npmignore
@@ -1,5 +1,5 @@
 **/*
 README.md
 LICENSE
-!lib/*
+!lib/**/*.js
 !cli.js

--- a/packages/playwright-mcp/package.json
+++ b/packages/playwright-mcp/package.json
@@ -24,12 +24,12 @@
     "./package.json": "./package.json"
   },
   "dependencies": {
+    "@modelcontextprotocol/sdk": "^1.6.1",
     "commander": "^13.1.0",
     "playwright": "1.52.0-next",
     "zod-to-json-schema": "^3.24.4"
   },
   "devDependencies": {
-    "@modelcontextprotocol/sdk": "^1.6.1",
     "@types/node": "^22.13.10"
   },
   "bin": {

--- a/packages/playwright-mcp/src/context.ts
+++ b/packages/playwright-mcp/src/context.ts
@@ -60,6 +60,6 @@ export class Context {
       url.searchParams.set('launch-options', JSON.stringify(this._launchOptions));
       return await playwright.chromium.connect(String(url));
     }
-    return await playwright.chromium.launch(this._launchOptions);
+    return await playwright.chromium.launch({ channel: 'chrome', ...this._launchOptions });
   }
 }

